package.lisp

```diff
@@ -2051,7 +2040,18 @@
    ;; CLIM-EXTENSIONS
    #:medium-miter-limit
    #:medium-draw-glyph
-   #:medium-draw-circle*))
+   #:medium-draw-circle*
+   ;; ellipses
+   #:reparameterize-ellipse
+   #:ellipse-point
+   #:ellipse-point*
+   #:ellipse-derivative
+   #:ellipse-cubic-bezier-control-points
+   #:ellipse-cubic-bezier-control-points*
+   #:ellipse-cubic-bezier-points
+   #:ellipse-cubic-bezier-points*
+   #:transformation-angle
+   #:transform-angle))
```

Comment:
this is still wrong. clim-backend package exports symbols, which are meant to be implemented by a backend to be used from the core (so climb client is core module and provider is backend, not the other way around). Backend should import directly from climi.